lesson_08/test_homeworks08.py

```diff
@@ -7,3 +7,100 @@
 На оцінку впливає як якість тестів так і розмір тестового покриття.
 Мінімум на 10 балів - 1 правильно задизайнений позитивний тест на функцію.
 """
+
+import unittest
+import homeworks
+import warnings
+
+class Mytests(unittest.TestCase):
+
+    def test1_trees_1(self):
+        input_apple = 12
+        input_peer = 12
+        input_plum = 12
+        expected = 36
+        actual_result = homeworks.trees_total(input_apple, input_peer, input_plum)
+        self.assertEqual(expected, actual_result, msg=f"Trees test 1 is FAILED")
+
+    def test2_trees_2(self):
+        input_apple = -12
+        input_peer = 105
+        input_plum = -8
+        expected = input_plum + input_peer + input_apple
+        actual_result = homeworks.trees_total(input_apple, input_peer, input_plum)
+        self.assertNotEqual(expected, actual_result, msg=f"Trees test 2 is FAILED")
+
+    def test3_trees_3(self):
+        input_apple = 0
+        input_peer = 0
+        input_plum = 0
+        expected = input_plum + input_peer + input_apple
+        actual_result = homeworks.trees_total(input_apple, input_peer, input_plum)
+        self.assertTrue(expected == actual_result, msg=f"Trees test 3 is FAILED")
+
+    def test4_text_1(self):
+        text = "Pfj   jfekk efk  jejfl            ejfkejhlkj;jorsg fehfqi  jhefqhk"
+        expected = 1
+        result = homeworks.space_delete(text)
+        if "  " in result:
```

Comment:
```suggestion
        if "  " in result:
```
вживати if  у тесті не дуже бажано